scripts/libs.versions.toml

```diff
@@ -3,28 +3,25 @@
 # Do not add a dependency to an individual sample, edit the global version instead.
 #####
 [versions]
-accompanist = "0.34.0"
-androidGradlePlugin = "8.5.0"
-androidx-activity-compose = "1.9.0"
+accompanist = "0.36.0"
+androidGradlePlugin = "8.6.0"
+androidx-activity-compose = "1.9.2"
 androidx-appcompat = "1.7.0"
 androidx-benchmark = "1.2.4"
 androidx-benchmark-junit4 = "1.2.4"
-androidx-compose-bom = "2024.06.00"
-androidx-compose-latest = "1.7.0-beta07"
-androidx-compose-material3 = "1.3.0-beta05"
-androidx-compose-material3-adaptive = "1.0.0-beta04"
```

Comment:
Where does the version for adaptive come from now? I am guessing the BOM, but I don't see it on the [mapping page](https://developer.android.com/develop/ui/compose/bom/bom-mapping)?

Reply:
It's in the BOM now! Just checked, we need to get the mapping page updated